proposals/lip-0037.md

```diff
@@ -67,26 +69,37 @@ The tag is chosen depending on the message type. Moreover, different serializati
 
 When a new tag is required in the future, the format:
 
-    "LSK_" || TAG || "_"
+    "LSK_" + TAG + "_"
 
 has to be used where:
 
 - Strings in double quotes are ASCII-encoded literals.
 - TAG is a unique string indicating the scheme and, optionally, additional information, e.g., to specify a version number as shown [above](#message-tags). TAG must contain only ASCII characters between 0x21 and 0x7e (inclusive), except that it must not contain underscore (0x5f).
 
+### Chain Identifiers
+
+Chain identifiers are 4-byte values. The first byte is set to `CHAIN_ID_PREFIX_MAINNET` for chains running in the mainnet network and to `CHAIN_ID_PREFIX_TESTNET` for chains running in the testnet network. The other 3 bytes are unique to that chain.
+
+The following table defines the chain-identifiers prefixes currently specified.
+
+| Name                            | Type  | Value| Description                                                 |
+|---------------------------------|-------|------|-------------------------------------------------------------|
+| `CHAIN_ID_PREFIX_MAINNET`       | bytes | 0x00 | Chain-identifiers prefix for chains running in the mainnet. |
+| `CHAIN_ID_PREFIX_TESTNET`       | bytes | 0x01 | Chain-identifiers prefix for chains running in the testnet. |
+
 ### Signing and Verifying with Ed25519
 
-Let [`Sign`](https://tools.ietf.org/html/rfc8032#section-5.1.6) and [`Verify`](https://tools.ietf.org/html/rfc8032#section-5.1.7) be the signing and verifying functions of Ed25519 as specified in [RFC 8032](https://tools.ietf.org/html/rfc8032). Then the signature for a binary message `m` and a secret key `sk` is generated by `signMessage(sk, tag, networkIdentifier, m)` as defined below. `tag` must be the correct message tag for `m` and `networkIdentifier` the correct network identifier for the chain. The resulting signature `sig` in combination with the message `m` and the matching public key `pk` is verified by `verifyMessageSig(pk, tag, networkIdentifier, m, sig)`.
```

Comment:
The very end of the paragraph:
`pk` is verified by `verifyMessageSig(pk, tag, networkIdentifier, m, sig)`. -> `pk` is verified by `verifyMessageSig(pk, tag, chainID, m, sig)`.